packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -54,7 +61,7 @@ const ContentContainer = ({
     <div {...customProps} className={contentLayoutClassNames}>
       {header && <div className={cx('header')}>{header}</div>}
       <div className={cx('main')}>
-        <Scroll className={cx('normalizer')} refCallback={scrollRefCallback}>
+        <Scroll className={cx('normalizer')} refCallback={scrollRefCallback} tabIndex={setFocusOnContent ? '0' : '-1'}>
```

Comment:
So when there are interactive elements and setFocusOnContent is true, scrollbar would also be part of navigation right?

Reply:
shouldn't it go through the first interactive element directly as discussed when the new prop is set ?

Reply:
It will go to first interactive element directly when the prop is not set to true